cfg/cis-1.6/master.yaml

```diff
@@ -714,7 +714,9 @@ groups:
       - id: 1.2.26
         text: "Ensure that the --request-timeout argument is set as appropriate (Automated)"
         audit: "/bin/ps -ef | grep $apiserverbin | grep -v grep"
-        type: manual
```

Comment:
Marking it as manual is expected according to the description in cis 1.6: 
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/485054/185783813-7ccd8388-0173-4617-a09e-62ce66d5d183.png">

Reply:
Here is the origin PR which change it to manual: https://github.com/aquasecurity/kube-bench/pull/874